composer.json

```diff
@@ -27,7 +27,7 @@
     },
     "require-dev": {
         "psr/log": "~1.0",
-        "phpcr/phpcr-api-tests": "2.1.5",
+        "phpcr/phpcr-api-tests": "dev-create_and_reference_nodes_in_transaction",
```

Comment:
change to 2.1.6 once https://github.com/phpcr/phpcr-api-tests/pull/165 is merged and tagged